src/l2_packet/l2_packet_zephyr.c

```diff
@@ -166,6 +166,21 @@ l2_packet_init(const char *ifname, const u8 *own_addr, unsigned short protocol,
 		goto fail;
 	}
 
+	/* Set the protocol-defined priority of all packets sent on this socket to 6
+	 * (NET_PRIORITY_IC, Internetwork control).
+	 * Network packet priority settings described in IEEE 802.1Q Annex I.1
+	 * The high priority (>= NET_PRIORITY_CA, 3) packet will be pushed directly to network driver
```

Comment:
This will only be true if `NET_TC_SKIP_FOR_HIGH_PRIO` is enabled, so, please adjust the comment or enable this by default. But none the less, setting high priority is a good idea esp. for rekeying.

Reply:
Thanks! 
I have adjusted the comments and added a check if NET_TC_SKIP_FOR_HIGH_PRIO is enabled.

Reply:
I was only recommending to modify the comments, independent of that option we can still set a high priority.

Reply:
OK, I see.